usr/src/uts/common/fs/zfs/dmu_zfetch.c

```diff
@@ -197,9 +200,11 @@ dmu_zfetch_stream_create(zfetch_t *zf, uint64_t blkid)
  * routines to create, delete, find, or operate upon prefetch streams.
  */
 void
-dmu_zfetch(zfetch_t *zf, uint64_t blkid, uint64_t nblks)
+dmu_zfetch(zfetch_t *zf, uint64_t blkid, uint64_t nblks, boolean_t data)
```

Comment:
I've added commend for that argument and renamed it from 'data' to 'fetch_data'.  I am not sure that I like reversing sense of that argument, since negative options can be potential source of confusion due to double negation.